source/crud/tests/unified/aggregate-allowdiskuse.json

```diff
@@ -0,0 +1,155 @@
+{
+  "description": "aggregate-allowdiskuse",
+  "schemaVersion": "1.0",
+  "createEntities": [
+    {
+      "client": {
+        "id": "client0",
+        "observeEvents": [
+          "commandStartedEvent"
+        ]
+      }
+    },
+    {
+      "database": {
+        "id": "database0",
+        "client": "client0",
+        "databaseName": "crud-tests"
+      }
+    },
+    {
+      "collection": {
+        "id": "collection0",
+        "database": "database0",
+        "collectionName": "coll0"
+      }
+    }
+  ],
+  "initialData": [
+    {
+      "collectionName": "coll0",
+      "databaseName": "crud-tests",
+      "documents": []
+    }
+  ],
+  "tests": [
+    {
+      "description": "Aggregate does not send allowDiskuse when value is not specified",
```

Comment:
Nit:  allowDiskuse -> allowDiskUse

in all three tests.

Reply:
Fixed, as well as a previous typo in the `find` test file.